.github/workflows/pipeline.yml

```diff
@@ -38,6 +38,7 @@ jobs:
         env:
           SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
           SONAR_HOST_URL: 'https://sonarcloud.io/'
+          continue-on-error: 'true'
```

Comment:
Why do we need to continue on error for Jacoco and Sonar?

Reply:
I think it would be better to continue pipeline and see all errors

Reply:
Removed continue on error in jacoco step